src/components/routes/Settings/Settings.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 //@ts-ignore
-import { Button, Header, Modal, Form, TextArea, Icon } from 'semantic-ui-react';
+import { Button, Form, Header, Icon, Modal, TextArea } from 'semantic-ui-react';
```

Comment:
minor note: Settings isn't really its own route, it's just a modal that can be rendered on any page. This is fine for now, can reorganize later. It's also not currently enabled.